### DISCUSSION TITLE
How to Install Remote-SSH Extension on HA VSCode Add-on

### DISCUSSION THREAD

**Discussion by shawnhaywood** [hassio-addons/addon-vscode#823] [General]
Hello everyone!  I'm using VSCode HA add-on to manage and edit my HA files.  I would like to also use it to access my homelab servers via SSH.  However, when I search the VSCode extensions I can't find Remote-SSH.  I see other ssh extensions, but want to use Microsoft's Remote-SSH.  Maybe I'm going something wrong, or I can't add extensions using the HA add-on?

**Comment by MitchTalmadge** [comment]
Can't really do that with code-server as far as I know. This is not a full-blown VSCode install, it's a fork of it made to run in a web browser. Certain extensions cannot run in a browser.

See https://github.com/coder/code-server/discussions/4462#discussioncomment-1602076

**Comment by nolsen311** [comment]
So likewise, no way to remote IN to this code-server...?
